src/main/webapp/js/chart-loader.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+google.charts.load('current', {packages: ['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/**
+ * Builds a chart element and adds it to the page.
+ */
+function drawChart(){
+  var sentiment_data = new google.visualization.DataTable();
```

Comment:
I realize that you're probably just following what the guide told you to do, but fwiw the style guide recommends using let or const instead of var: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let